impeller/aiks/aiks_unittests.cc

```diff
@@ -246,6 +246,16 @@ TEST_P(AiksTest, CanRenderCurvedStrokes) {
   ASSERT_TRUE(OpenPlaygroundHere(canvas.EndRecordingAsPicture()));
 }
 
+TEST_P(AiksTest, CanRenderThickCurvedStrokes) {
```

Comment:
May I have some advice? Not entirely sure why this produce clipped golden
![image](https://github.com/flutter/engine/assets/47866232/47e37dbf-dbd8-4753-a1d5-ef2bbf458a81)
locally it runs fine